components/http.js

```diff
@@ -92,20 +91,20 @@ SteamCommunity.prototype._checkHttpError = function(err, response, callback, bod
 	}
 
 	if (response.statusCode >= 300 && response.statusCode <= 399 && response.headers.location.indexOf('/login') != -1) {
-		err = new Error("Not Logged In");
+		err = new Error('Not Logged In');
 		callback(err, response, body);
 		this._notifySessionExpired(err);
 		return err;
 	}
 
-	if (response.statusCode == 403 && typeof response.body === 'string' && response.body.match(/<div id="parental_notice_instructions">Enter your PIN below to exit Family View.<\/div>/)) {
-		err = new Error("Family View Restricted");
+	if (response.statusCode == 403 && typeof response.body == 'string' && response.body.match(/<div id="parental_notice_instructions">Enter your PIN below to exit Family View.<\/div>/)) {
```

Comment:
What if the response is not in English?

Reply:
A `Steam_Language=english` cookie is set in the constructor.